packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -543,7 +559,18 @@ ${summary.join('\n')}
         outputPath: _vmcodeOutputPath,
         workingDirectory: _buildDirectory,
         kernel: newestAppDill().path,
+        dumpDebugInfoPath: dumpDebugInfo ? _debugInfoOutpath : null,
       );
+
+      if (dumpDebugInfo) {
+        final debugDirectory = Directory(_debugInfoOutpath)
```

Comment:
I was thinking we'd first create a temp directory and point aot tools to the temp directory for the debug output and then we'd zip the temp directory and copy it into the build directory. What do you think?